com.unity.ml-agents/Runtime/Inference/TensorApplier.cs

```diff
@@ -52,16 +52,25 @@ public interface IApplier
             Dictionary<int, List<float>> memories,
             object barracudaModel = null)
         {
-            actionSpec.CheckNotHybrid();
-
+            bool useDeprecated = false;
+            if (barracudaModel != null)
+            {
+                var model = (Model)barracudaModel;
+                useDeprecated = model.UseDeprecated();
+            }
+            if (useDeprecated)
+            {
+                actionSpec.CheckNotHybrid();
+            }
             if (actionSpec.NumContinuousActions > 0)
             {
-                m_Dict[TensorNames.ActionOutput] = new ContinuousActionOutputApplier();
+                var tensorName = useDeprecated ? TensorNames.ActionOutputDeprecated : TensorNames.ContinuousActionOutput;
```

Comment:
`model.ContinuousOutputName()`?

Reply:
Looks like it's possible that the model is `null` here